marklogic-data-hub-central/ui/src/pages/Run.tsx

```diff
@@ -398,6 +398,7 @@ const Run = (props) => {
                 setOpenJobResponse={setOpenJobResponse}
                 isStepRunning={isStepRunning}
                 canUserStopFlow={userCanStopFlow}
+                isLoading={flows.length === 0 || steps.length === 0}
```

Comment:
@littlecherro I'm seeing on an established project that the spinner looks good. But my concern with this logic here is that on a fresh project with no steps or flows created yet, will it be spinning forever? Ideally maybe the spinner should be attached to the API calls?

Reply:
yeah! how could i forgot that case 👀 

working on that